castle/cms/cron/_twitter_monitor.py

```diff
@@ -82,7 +81,6 @@ def parse_line(site, public_url, line):
                 record_tweet_stats(site, ob, tweet)
     if modified:
         transaction.commit()
-        index_in_es()
```

Comment:
my comment for the same change in the _social_counts.py file applies here -- this should be reverted for now, and the import location updated.